torch/quantization/fx/quantize.py

```diff
@@ -318,6 +318,60 @@ def node_arg_is_bias(node: Node, arg: Any) -> bool:
                     return True
     return False
 
+def all_node_args_have_no_tensors(node: Node) -> bool:
```

Comment:
in the short term, this sounds OK.  In the longer term, we might need to evaluate introducing an output type for nodes, which is known for some nodes (and maybe not others) and can propagate through various ops.  For example, we know that the output of `x.size(...)` is always an integer, etc.